Microsoft.OneWeek.Hack.Microids.MessageRouter/Program.cs

```diff
@@ -46,16 +46,31 @@ public static async Task Main(string[] args)
                 {
                     // Get a new message router object.
                     var messagerouter = serviceProvider.GetService<EnrichmentMessageRouter>();
+
+                    var httpClient = new System.Net.Http.HttpClient();
+                    var resp = await httpClient.GetAsync("https://www.microsoft.com");
+                    Console.WriteLine(await resp.Content.ReadAsStringAsync());
+
                     messagerouter.Initiate(cts.Token);
                     await WhenCancelled(cts.Token);
                 }
             }
         }
 
+        private static string AppInsightsKey
```

Comment:
We can do this in a separate checkin but we can set up the config to pull environment variables and use IConfiguration to pull settings to avoid taking a hard dependency on the environment.